plugin/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 
 dependencies {
     implementation("com.android.tools.build:gradle:3.0.1")
-    implementation("com.google.apis:google-api-services-androidpublisher:v3-rev6-1.23.0") {
+    implementation("com.google.apis:google-api-services-androidpublisher:v3-rev11-1.23.0") {
```

Comment:
They've already pumped out `rev12` 😆. I say we update it this time and then just stop for a long while. Otherwise, we'll constantly be updating it.

Reply:
Yeah, I updated it and we should probably delay the constant update to the releases..?

Reply:
SGTM